### ISSUE TITLE
danger local fails to find token in local environment

### ISSUE THREAD

**Issue by garnett** [danger/danger#758]
# Report

## What did you do?

`danger local` in my feature branch

## What did you expect to happen?

I see dry run of Danger for this branch (or I don't understand `local` option)

## What happened instead?

```
Running your Dangerfile against this PR - https://github.com/xxx/pull/2333
Turning on --verbose

Local repository was not found on GitHub. If you're trying to test a private repository please provide a valid API token through DANGER_GITHUB_API_TOKEN environment variable.
```

Even though:
`echo $DANGER_GITHUB_API_TOKEN` prints the token

## Your Environment

danger 4.3.0

**Comment by php-coder** [comment]
Yes, it's frustrating :-( Unfortunately I had a dig into the Rube code because there is no other way to understand what's going on.

**Comment by orta** [comment]
Maybe that's a bug? I don't work on any private repos, so it's possible that it's something I've missed

**Comment by rev112** [comment]
`danger local` works fine for me when executed against private repos.
@garnett How do you define your `DANGER_GITHUB_API_TOKEN` variable? Perhaps you don't export it as an environment variable?
For example, `export DANGER_GITHUB_API_TOKEN=123` is how it's done in `bash`.

**Comment by pboling** [comment]
What specific permission should this token have @orta
